### PR TITLE
Navmazing for settings

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: cfme/tests/configure/test_visual_infra.py -v -k 'test_grid_page_per_item'}}

Purpose or Intent
=================

Navmazing for settings.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 13fd897c715d5614ecd642b23b3d85c96e79f9e4:

`cfme/configure/settings.py`:
- :bangbang: Line 53:101: **E501** *line too long (107 > 100 characters)*
- :bangbang: Line 106:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 113:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 120:1: **E302** *expected 2 blank lines, found 1*
- :red_circle: Line 240:9: **F821** *undefined name 'snavigate_to'*
- :bangbang: Line 317:1: **E302** *expected 2 blank lines, found 1*

`cfme/tests/configure/test_display_settings.py`:
- :red_circle: Line 70:21: **F821** *undefined name 'Visual'*
- :red_circle: Line 83:17: **F821** *undefined name 'Visual'*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a1dbf4250ca9950a9538e0698ebbfb6020d86e46:

`cfme/configure/settings.py`:
- :heavy_exclamation_mark: Line 53:91: **W291** *trailing whitespace*
- :bangbang: Line 322:1: **E302** *expected 2 blank lines, found 1*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 56487945c945a447600bb774a1ce049a98f22383:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/configure/settings.py [hunk 1]
```diff
@@ -24,27 +27,13 @@
...
-                "timeprofile_edit":
-                lambda ctx: timeprofile_table.click_cell(
-                    "description", ctx.timeprofile.description),
-
-            }
-        ],
-        'my_settings_visual': [lambda _: tabs.select_tab("Visual"), {}],
-        'my_settings_default_filters': [lambda _: tabs.select_tab("Default Filters"), {}],
+
         'my_settings_default_views': [lambda _: tabs.select_tab("Default Views"), {}],
```
Comment (izapolsk) [inline cfme/configure/settings.py]: Would you mind me asking if you intend to replace this force_nav branch as well in this PR ?

File: cfme/base/ui.py [hunk 2]
```diff
@@ -59,6 +59,19 @@ def step(self):
 
 
 @navigator.register(Server)
+class MySettings(CFMENavigateStep):
+    prerequisite = NavigateToSibling('LoggedIn')
+
+    def step(self):
+        if self.obj.appliance.version > '5.7':
```
Comment (izapolsk) [inline cfme/base/ui.py]: I'd use >= here because such a string comparison isn't obvious.

File: cfme/configure/settings.py [hunk 3]
```diff
@@ -24,27 +27,13 @@
...
     }
 )
 
 
-class Timeprofile(Updateable):
+class Timeprofile(Updateable, Navigatable):
```
Comment (izapolsk) [inline cfme/configure/settings.py]: It would be good to rename this class to CamelCase according to PEP8.

File: cfme/configure/settings.py [hunk 4]
```diff
@@ -114,7 +105,31 @@ def delete(self):
             'Time Profile "{}": Delete successful'.format(self.description))
 
 
-class Visual(Updateable):
+@navigator.register(Timeprofile, 'All')
+class TimeprofileAll(CFMENavigateStep):
```
Comment (izapolsk) [inline cfme/configure/settings.py]: Classes are usually named in CamelCase in Python.

File: cfme/configure/settings.py [hunk 5]
```diff
@@ -114,7 +105,31 @@ def delete(self):
...
+    prerequisite = NavigateToAttribute('appliance.server', 'MySettings')
+
+    def step(self):
+        tabs.select_tab("Time Profiles")
```
Comment (izapolsk) [inline cfme/configure/settings.py]: it would be good to add appropriate am_I_here methods where it is needed.

**Comment by akarol** [comment]
@izapolsk I will consider all this changes may be in the next PR!
Thanks!

**Pull request merged**